pkg/cluster/manager/display.go

```diff
@@ -44,6 +45,7 @@ type InstInfo struct {
 	Ports     string `json:"ports"`
 	OsArch    string `json:"os_arch"`
 	Status    string `json:"status"`
+	Uptime    string `json:"uptime"`
```

Comment:
I think it's better be "Since" in the table header, so no extra documents about what does it mean if the node is down will be ever needed.

Reply:
Thanks for the advice, `Status` + `Since` forms the component's full information.

Reply:
I've added an argument `--uptime`, not `--since`, the latter seems weird. 🤔